data/src/pixel.rs

```diff
@@ -260,7 +358,44 @@ fn align(v: usize, a: usize) -> usize {
     (v + a - 1) & !(a - 1)
 }
 
+// Constructor funcions are const because they are used in other constructor evaluator.
+// Es. Formats
 impl Chromaton {
+    pub const fn new(
```

Comment:
Could you please spin the constructors away in another PR?

 I'd merge this part today.